diff --git a/kernel/conf/account.go b/kernel/conf/account.go
index c4590e22e..133208f71 100644
--- a/kernel/conf/account.go
+++ b/kernel/conf/account.go
@@ -23,7 +23,7 @@ type Account struct {
 
 func NewAccount() *Account {
 	return &Account{
-		DisplayTitle: true,
-		DisplayVIP:   true,
+		DisplayTitle: false,
+		DisplayVIP:   false,
 	}
 }
diff --git a/kernel/conf/appearance.go b/kernel/conf/appearance.go
index 908de478f..c403c1587 100644
--- a/kernel/conf/appearance.go
+++ b/kernel/conf/appearance.go
@@ -50,8 +50,8 @@ func NewAppearance() *Appearance {
 		Icon:                "litheness",
 		CodeBlockThemeLight: "github",
 		CodeBlockThemeDark:  "base16/dracula",
-		Lang:                "en",
-		CloseButtonBehavior: 0,
+		Lang:                "zh_CN",
+		CloseButtonBehavior: 1,
 		HideToolbar:         true,
 		HideStatusBar:       false,
 		StatusBar:           &util.StatusBar{},
diff --git a/kernel/conf/sync.go b/kernel/conf/sync.go
index b076c5e7e..662b300d4 100644
--- a/kernel/conf/sync.go
+++ b/kernel/conf/sync.go
@@ -38,8 +38,8 @@ func NewSync() *Sync {
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderS3,
 		Interval:            30,
 		LAN:                 &LANSync{MaxConcurrentReqs: 16},
 	}
