diff --git forkSrcPrefix/kernel/model/updater.go forkDstPrefix/kernel/model/updater.go
index f902b49d6b32d35318eb0b237037f2b4f79b8d33..cdbbc35027afb7c1173056105065268d238f68e2 100644
--- forkSrcPrefix/kernel/model/updater.go
+++ forkDstPrefix/kernel/model/updater.go
@@ -306,6 +306,11 @@ func skipNewVerInstallPkg() bool {
 	}
 	if !Conf.System.DownloadInstallPkg {
 		return true
+	} else {
+		// Conf.System.DownloadInstallPkg(自动下载更新安装包) 强制赋值为false
+		Conf.System.DownloadInstallPkg = false
+		Conf.Save()
+		return true
 	}
 	if gulu.OS.IsWindows() {
 		plat := strings.ToLower(Conf.System.OSPlatform)
