diff --git a/kernel/api/system.go b/kernel/api/system.go
index 93f5876e9..31df9aaf2 100644
--- a/kernel/api/system.go
+++ b/kernel/api/system.go
@@ -412,16 +412,17 @@ func normalizeCustomEmojiPath(name, ext string) (string, error) {
 }
 
 func checkUpdate(c *gin.Context) {
-	ret := gulu.Ret.NewResult()
-	defer c.JSON(http.StatusOK, ret)
+	// ret := gulu.Ret.NewResult()
+	// defer c.JSON(http.StatusOK, ret)
 
-	arg, ok := util.JsonArg(c, ret)
-	if !ok {
-		return
-	}
+	// arg, ok := util.JsonArg(c, ret)
+	// if !ok {
+	// 	return
+	// }
 
-	showMsg := arg["showMsg"].(bool)
-	model.CheckUpdate(showMsg)
+	// showMsg := arg["showMsg"].(bool)
+	// model.CheckUpdate(showMsg)
+	return
 }
 
 func exportLog(c *gin.Context) {
diff --git a/kernel/conf/system.go b/kernel/conf/system.go
index 8bf0031d2..8fd9ba567 100644
--- a/kernel/conf/system.go
+++ b/kernel/conf/system.go
@@ -77,7 +77,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
-		DownloadInstallPkg: true,
+		DownloadInstallPkg: false,
 		UpdateChannel:      UpdateChannelStable,
 	}
 }
diff --git a/kernel/model/mount.go b/kernel/model/mount.go
index dba3aff7d..9b491b9e4 100644
--- a/kernel/model/mount.go
+++ b/kernel/model/mount.go
@@ -440,8 +440,8 @@ func mountBox(boxID string) (alreadyMount bool, err error) {
 		task.AppendAsyncTaskWithDelay(task.PushMsg, 3*time.Second, util.PushErrMsg, Conf.Language(244), 7000)
 		go func() {
 			// 每次打开帮助文档时自动检查版本更新并提醒 https://github.com/siyuan-note/siyuan/issues/5057
-			time.Sleep(time.Second * 10)
-			CheckUpdate(true)
+			// time.Sleep(time.Second * 10)
+			// CheckUpdate(true)
 		}()
 	}
 
diff --git a/kernel/model/updater.go b/kernel/model/updater.go
index 8a39a626b..6db50b966 100644
--- a/kernel/model/updater.go
+++ b/kernel/model/updater.go
@@ -267,6 +267,10 @@ func isVersionUpToDate(releaseVer string) bool {
 var skipInstallPkgPlatformCached = -1
 
 func skipNewVerInstallPkg() bool {
+	// Conf.System.DownloadInstallPkg(自动下载更新安装包) 强制赋值为false
+	Conf.System.DownloadInstallPkg = false
+	Conf.Save()
+
 	if skipInstallPkgPlatformCached == -1 {
 		skipInstallPkgPlatformCached = 0
 		if !gulu.OS.IsWindows() && !gulu.OS.IsDarwin() {
